crates/agglayer/src/cli.rs

```diff
@@ -47,7 +47,7 @@ pub(crate) enum Commands {
 
     Prover {
         /// The path to the configuration file.
-        #[arg(long, short, value_hint = ValueHint::FilePath, default_value = "agglayer-prover.toml", env = "PROVER_CONFIG_PATH")]
+        #[arg(long, short, value_hint = ValueHint::FilePath, default_value = "agglayer-prover.toml", env = "PROVER_AGGLAYER_CONFIG_PATH")]
```

Comment:
I'd say let's have all the Agglayer-related env variables start with `AGGLAYER_` so they can be seen easily in an alphabetic listing of all env variables.
```suggestion
        #[arg(long, short, value_hint = ValueHint::FilePath, default_value = "agglayer-prover.toml", env = "AGGLAYER_PROVER_CONFIG_PATH")]
```